format/opentimestamps/ots.go

```diff
@@ -0,0 +1,123 @@
+package opentimestamps
+
+import (
+	"encoding/binary"
+	"errors"
+	"io"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+func init() {
+	interp.RegisterFormat(
+		format.OpenTimestamps,
+		&decode.Format{
+			Description:  "OpenTimestamps file",
+			Dependencies: nil,
```

Comment:
To make the format probeable add it to the probe group `Groups: []*decode.Group{format.Probe},`. fq by design currently do not have any special per format code for probing it's up to a format decoder to fail early which seems to be the case here